pyfolio/plotting.py

```diff
@@ -573,6 +574,24 @@ def plot_rolling_returns(
         The axes that were plotted on.
 
 """
+    def draw_cone(returns, num_stdev, live_start_date, ax):
+        cone_df = timeseries.cone_rolling(
+            returns,
+            num_stdev=num_stdev,
+            cone_fit_end_date=live_start_date)
+
+        cone_df_fit = cone_df[cone_df.index < live_start_date]
+
+        cone_df_live = cone_df[cone_df.index > live_start_date]
+        temp_keep_days = cone_df_live.index < returns.index[-1]
```

Comment:
I find this double-slicing with a temp variable a bit confusing. Can we combine that to a single slice with an `&`?